deployment/terraform/assets/provisioners/agent.sh

```diff
@@ -14,6 +14,12 @@ do
         sudo apt-get -y update && \
         sudo apt-get install -y prometheus-node-exporter && \
         sudo apt-get install -y numactl linux-tools-aws linux-tools-aws-lts-22.04 && \
+        # Install OpenTelemetry collector, using ubuntu user to avoid permission issues
+        wget https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v0.110.0/otelcol-contrib_0.110.0_linux_amd64.deb && \
+        sudo dpkg -i otelcol-contrib_0.110.0_linux_amd64.deb && \
+        sudo sed -i 's/User=.*/User=ubuntu/g' /lib/systemd/system/otelcol-contrib.service && \
+        sudo sed -i 's/Group=.*/Group=ubuntu/g' /lib/systemd/system/otelcol-contrib.service && \
```

Comment:
This looks brittle. Would it be worth having our own service file and just copy-pasting it over to the instance? 0/5

Reply:
I'd say that having our own service file copied brings other forms of brittleness to the table, so I'd keep this as is.